src/NativeScript/GlobalObject.mm

```diff
@@ -286,13 +286,15 @@ static EncodedJSValue JSC_HOST_CALL collectGarbage(ExecState* execState) {
     }
     }
 
-    if (symbolWrapper) {
-        object->putDirectWithoutTransition(vm, propertyName, symbolWrapper);
-        propertySlot.setValue(object, None, symbolWrapper);
+    if (!symbolWrapper) {
+        throwTypeError(execState, WTF::String::format("Metadata for \"%s\" found, but symbol \"%s\" is not available at runtime.", symbolMeta->jsName(), symbolMeta->name()));
```

Comment:
Is this going to break our [ApiIterator test](https://github.com/NativeScript/ios-runtime/blob/master/tests/TestRunner/app/ApiTests.js#L443) because of some symbols which doesn't exist at runtime but there is metadata for them? If yes, we can add try-catch block in [ApiIterator test](https://github.com/NativeScript/ios-runtime/blob/master/tests/TestRunner/app/ApiTests.js#L443).